ts/session/utils/Groups.ts

```diff
@@ -10,11 +14,16 @@ export async function getGroupMembers(groupId: PubKey): Promise<Array<PubKey>> {
     return [];
   }
 
-  return groupMembers.map((member: string) => new PubKey(member));
+  const promises = (groupMembers as Array<string>).map(async (member: string) =>
+    MultiDeviceProtocol.getPrimaryDevice(member)
+  );
+  const primaryDevices = await Promise.all(promises);
+
+  return _.uniqWith(primaryDevices, (a, b) => a.isEqual(b));
 }
 
-export function isMediumGroup(groupId: PubKey): boolean {
-  const conversation = window.ConversationController.get(groupId.key);
+export function isMediumGroup(groupId: string): boolean {
```

Comment:
Why string?

Reply:
As above a closed group id is not a public key, a medium group id might be but we should handle all group cases